src/main/java/at/hannibal2/skyhanni/data/mob/MobDetection.kt

```diff
@@ -313,8 +302,7 @@ class MobDetection {
             // is S0EPacketSpawnObject -> addEntityUpdate(packet.entityID)
             is S37PacketStatistics -> // one of the first packets that is sent when switching servers inside the BungeeCord Network (please some prove this, I just found it out via Testing)
             {
-                shouldClear.set(true)
-                allEntitiesViaPacketId.clear()
+                clearMobs()
```

Comment:
ehm, you break thread safety here. Will lead to an ConcurrentModifactionExpection.